src/main/config/secor.common.properties

```diff
@@ -139,6 +139,14 @@ kafka.seed.broker.port=9092
 # to as the chroot.
 kafka.zookeeper.path=/
 
+# Store offset in zookeeper and kafka consumer topic.
+# Possible values: true or false
+kafka.dual.commit.enabled=true
```

Comment:
Also indicate that this parameter is only used when offsets.storage is set to 'kafka', and give a link to kafka configuration doc.